src/main/java/muramasa/antimatter/materials/MaterialType.java

```diff
@@ -106,9 +107,25 @@ public int getLayers() {
         return tagMap.get(this);
     }
 
-    //TODO
+    public Item get(Material material) {
+        if (!allowGeneration(material)) Utils.onInvalidData("GET ERROR - DOES NOT GENERATE: T(" + id + ") M(" + material.getId() + ")");
+        ItemStack replacement = AntimatterAPI.getReplacement(this, material);
+        if (!replacement.isEmpty()) return replacement.getItem();
+        MaterialItem item = AntimatterAPI.get(MaterialItem.class, id + "_" + material.getId());
+        if (item == null) Utils.onInvalidData("GET ERROR - MAT ITEM NULL: T(" + id + ") M(" + material.getId() + ")");
+        return item;
+    }
+
     public ItemStack get(Material material, int count) {
-        return ItemStack.EMPTY;
```

Comment:
This method can just call the method above to get the item, instead of repeating code